docs/jitserver_tuning.md

```diff
@@ -44,9 +44,12 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). The default is to use a nameless cache.
 
+ By default, the amount of memory that an AOT cache can consume at the server is not limited. The number of caches that a JITServer can hold is also not limited.
```

Comment:
The default is 300 MB, not unlimited.

Reply:
How about "The number of caches that a JITServer can hold is also not limited."? Is this still true?

Reply:
Yes, that is still true. What matters is the memory consumption. If we limit that, the number of caches will be limited implicitly as each cache consumes memory.